### PR TITLE
Add new tables

### PR DESCRIPTION
**Pull request by Ukaypromise**
### Query Multiple Tables

- Create a table named owners with the following columns.
- Create a table named species with the following columns.
- Modify animals table:
image1
image2

- Insert the following data into the owners table
- Insert the following data into the species table
image3

- Modify your inserted animals so it includes the species_id value:
image4

- Modify your inserted animals to include owner information (owner_id)

image5

image6

- Write queries (using JOIN) to answer the following questions:

1. What animals belong to Melody Pond?
2. List of all animals that are pokemon (their type is Pokemon).
3. List all owners and their animals, remember to include those that don't own any animal.
4. How many animals are there per species?
5. List all Digimon owned by Jennifer Orwell.
6. List all animals owned by Dean Winchester that haven't tried to escape.
7. Who owns the most animals?

image8
image9
image10
image11
image12
image13
image 7

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -9,3 +9,14 @@ CREATE TABLE animals (
...
+CREATE TABLE owners (
+    id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    full_name VARCHAR(250),
+    age INT,    
+);
+
+CREATE TABLE species (
+    id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(250),   
+);
```
Comment (Whoistolu) [inline schema.sql]: - [x] [REQUIRED] You have done a great job adding the `owners` and `species` tables, however, kindly note that part of the project requirements specify that you should modify the `animals` table.
1. Remove column species
2. Add column species_id which is a foreign key referencing `species` table
3. Add column owner_id which is a foreign key referencing the `owners` table

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @Ukaypromise 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Hightlights

✔️ Meaningful Commit Messages
✔️ Descriptive PR

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (Ukaypromise) [reply schema.sql]: Thank you @Whoistolu for bringing my attention to this. I have swiftly updated the schema file.

**Review by andersonlebon — APPROVED** [review]
## Hello @Ukaypromise 
# STATUS: APPROVED! 🥇 💯
unnamed

✅ Good pull request description 👍
✅ Good commit messages 👏

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

___ 

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

**Pull request merged**